docs/source/test_network.md

```diff
@@ -455,13 +452,42 @@ Creating ca_org1    ... done
 Creating ca_orderer ... done
 ```
 
-The script then uses the Fabric CA client to register the users that belong to
-each organization and generate the certificates and keys for each identity. You
-can find the commands that are used to set up the network in the `registerEnroll.sh`
-script in the `organizations/fabric-ca` directory. To learn more about how you
-would use the Fabric CA to deploy a Fabric network, visit the
-[Fabric CA operations guide](https://hyperledger-fabric-ca.readthedocs.io/en/latest/operations_guide.html).
-You can learn more about how Fabric uses PKI by visiting the [identity](identity/identity.html) and [membership](membership/membership.html) concept topics.
+It is worth taking time to examine the logs generated by the `./network.sh`
+script after the CAs have been deployed. The test network uses the Fabric CA
+client to register node and user identities with the CA of each organization. The
+script then uses the enroll command to generate an MSP folder for each identity.
+The MSP folder contains the certificate and private key for each identity, and
+establishes the identity's role and membership in the organization that operated
+the CA. You can use the following command to examine the MSP folder of the Org1
+admin user:
+```
+tree organizations/peerOrganizations/org1.example.com/users/Admin/@org1.example.com/
+```
+The command will reveal the MSP folder structure and configuration file:
+```
+organizations/peerOrganizations/org1.example.com/users/Admin@org1.example.com/
+└── msp
+    ├── IssuerPublicKey
+    ├── IssuerRevocationPublicKey
+    ├── cacerts
+    │   └── localhost-7054-ca-org1.pem
+    ├── config.yaml
+    ├── keystore
+    │   └── 58e81e6f1ee8930df46841bf88c22a08ae53c1332319854608539ee78ed2fd65_sk
+    ├── signcerts
+    │   └── cert.pem
+    └── user
+```
+You can learn more about the contents of MSP folders by going to the [Membership Service Provider](membership/membership.html)
+concept topic.
+
+Both cryptogen and the Fabric CAs generate the crypto material for each organization
```

Comment:
This is a good addition to the topic - a nice gentle intro to MSPs.

To tie it all together I think it would be worth also pointing out that the certificate, also known as the signing cert or public key is in the signcerts folder and the private key is in the keystore folder.

Reply:
Probably not for mentioning in this particular doc, but at some point (maybe in the msp topic) we can also mention that renaming certs and keys is also possible to make them easier to manage.

Reply:
Thanks @pamandrejko. I am going to side with @denali49 on this point in the interest of keeping this topic short. Hopefully this topic serves to guide users to other parts of the docs.